back-end/requests/put/add_members.js

```diff
@@ -0,0 +1,43 @@
+let Playlist = require("../../models/playlists.model");
+
+const add_member = async (req, res) => {
+  const group_id = req.params.group_id
+  const user_id = req.params.user_id
+
+  let members
+  let banned_members
+  await Playlist.findOne({_id:group_id}) //retrieve the group information
+    .then(response => {
+      members = response.members
+      banned_members = response.banned_members
+    })
+    .catch(err => {
+      const msg = "Cannot find the group" 
+      console.log(msg)
+      return new Error(msg)
+    })
+  if (members.includes(user_id))  //check if in group already
+  {
+    const msg = "User already in the group"
+    console.log(msg)
+    return new Error(msg)
```

Comment:
we want to pass this new error to middleware, so we should really write `return next(new Error(msg))`